### PR TITLE
Merge changes from 07x to 08x

### PR DESCRIPTION
**Pull request by stan-confluent**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def run_tests(self):
                         'paramiko~=2.7.2',
                         'pyzmq==19.0.2',
                         'pycryptodome==3.9.8',
+                        'bcrypt==3.1.7',
+                        'pysistence==0.4.1',
```
Comment (andrewegel) [inline setup.py:59]: Don't need this since 0.8.x is Py3x compatible, these pins were for Py2.7

**Review by andrewegel — CHANGES_REQUESTED** [review]
Undo the bcrypt / pysistence pins, those were for Py2.7 compatibility

File: setup.py (hunk 1 above)
Comment (stan-confluent) [reply setup.py:59]: Do we officially state that 0.8.x drops support for 2.7? I'm all in favor for doing so for the record :)

**Comment by stan-confluent** [comment]
@andrewegel done

File: setup.py (hunk 1 above)
Comment (andrewegel) [reply setup.py]: 0.7.x line will be the last Py2.7 compatibility branch. 0.8.x is for python3. "Official" wording and enforcement is something else (You would want to set `python_requires >= 3.6` in setup.py to "officially" drop Py2.7 support) - And likely some kind of documentation note.

**Review by andrewegel — APPROVED** [review]

**Pull request merged**